components/Header.js

```diff
@@ -0,0 +1,15 @@
+import Link from 'next/link';
+
+export default function Header() {
```

Comment:
Maybe it would make sense to rename the function and the file to Navigation( ), as this is essentially what you are doing here :)
If you do, don't forget to also rename it in the Layout component :p

Reply:
Renamed :)